### DISCUSSION TITLE
import svg as pdf_tex

### DISCUSSION THREAD

**Discussion by AurelWeinhold** [typst/typst#2903] [Q&A] [answered]
Hi, I'm trying to import a svg that includes some text into typst. In LaTeX I could export the svg as pdf with the text omitted and in a separate tex file, i.e. a .pdf_tex, using inkscape. This pdf_tex I could then import into the document to have the text in the svg set by the latex compiler. Is this somehow possible in typst as well?

**Comment by Enivex** [comment, marked as answer]
Sure, if someone makes typst export for Inkscape.

Not exactly what you're asking for, but https://github.com/textext/textext might interest you

(You could also consider using cetz)
